lib/image_properties.dart

```diff
@@ -15,6 +15,7 @@ class ImageProperties {
   final bool matchTextDirection;
   final FilterQuality filterQuality;
   final double scale;
+  final String prefixNetworkImageRelativePath;
```

Comment:
I think I would prefer `prefixImagePath` for the name of this property, since it is implied that we mean a network image, and people may use this property for things other than relative paths.